src/arguments.rs

```diff
@@ -25,7 +25,9 @@ pub enum Command {
         #[structopt(short, long)]
         description: String,
         #[structopt(short, long)]
-        project: String,
+        project: Option<String>,
+        #[structopt(short, long)]
+        billable: Option<bool>,
```

Comment:
I did some checking on one of my branches but I think you can go with just use `bool` directly here
```suggestion
        billable: bool,
```
here and it'll default to `false`, not be a required flag.
Give that, what would be your preference?

Reply:
Neat, consider it done!